src/components/Herosection/hero.css

```diff
@@ -0,0 +1,53 @@
+.heroContainer {
+  width: 100%;
+  position: relative;
+  height: calc(100vh-80px);
+  /* background-color: blue; */
+}
+
+.heroBackgroundImgContainer {
+  width: 100%;
+  height: 90vh;
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url("./bgImage.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  object-fit: cover;
+  /* opacity: ; */
+}
+
+.heroTaglineContainer {
+  width: 50%;
+  padding: 0 2.1875rem;
+  padding-top: 2rem;
+  padding-bottom: 4.125rem;
+  width: 80%;
+  color: white;
+  text-shadow: 0 2px 3px rgba(0, 0, 0, 0.3);
+}
+
+@media (min-width: 768px) {
+  .heroTaglineContainer {
+    max-width: 50%;
+    padding-top: 7.5rem;
+    padding-bottom: 4.125rem;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+}
+.heroTaglineHeader {
+  margin-bottom: 10px;
+}
+.heroSubTagline {
+  margin-bottom: 57px;
+  line-height: 1.3;
+}
+.TaglineText {
```

Comment:
I suggest you try to be uniform with your className may be you should change that to camelCase